### PR TITLE
Add translation

### PR DESCRIPTION
**Pull request by kleingeist**
Translation for "maptopicprio" is still missing:

```
# grep -Pzo '\N+\nmsgstr \"\"\n\n' locale/de/LC_MESSAGES/django.po                                                                                            
msgid "Filter bar "
msgstr ""

msgid "Place Prioritization"
msgstr ""

msgid "Comment and prioritize places located on a map."
msgstr ""

msgid "Places"
msgstr ""

msgid "Locate the place on a map"
msgstr ""

msgid "Place label"
msgstr ""

msgid "Prioritize and comment places located on a map."
msgstr ""

msgid "place prioritization"
msgstr ""

msgid "Confirm Place Deletion"
msgstr ""

msgid "Are you sure you want to delete the place?"
msgstr ""

msgid "Add a new place"
msgstr ""

msgid "Add Place"
msgstr ""

msgid "Edit place"
msgstr ""

msgid "Places with comments"
msgstr ""

msgid "The place has been deleted"
msgstr ""
```

### CODE REVIEW

**Comment by kleingeist** [comment]
Note, that copyright fields are somehow not translated.
"Header image copyright", "Copyright shown in the header image"

**Comment by kleingeist** [comment]
added translations for the copyright field based on liqd/adhocracy4#182

File: locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -88,12 +88,12 @@ msgstr ""
 #: adhocracy4/images/fields.py:89
 #: adhocracy4/images/fields.py:89
 msgid "{} copyright"
-msgstr ""
+msgstr "{} Quellenangabe"
```
Comment (2e2a) [inline locale/de/LC_MESSAGES/django.po]: Unsure if Quellenangabe is the correct term here. Maybe: Urheberrecht.

File: locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -88,12 +88,12 @@ msgstr ""
...
 
 #: adhocracy4/images/fields.py:90
 #: adhocracy4/images/fields.py:90
 msgid "Copyright shown in the {}"
-msgstr ""
+msgstr "Quellenangabe die für {} angezeigt wird"
```
Comment (2e2a) [inline locale/de/LC_MESSAGES/django.po]: see above

File: locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -1120,19 +1120,19 @@ msgstr "Phaseneinstellungen bearbeiten"
 
 #: meinberlin/apps/dashboard2/dashboard.py:74
 msgid "Areasettings"
-msgstr ""
+msgstr "Bereich"
```
Comment (2e2a) [inline locale/de/LC_MESSAGES/django.po]: Suggestion: Karte

**Pull request merged**